README.md

```diff
@@ -103,215 +103,212 @@ test result: ok. 2 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out (4ms)
 
 ## Documentation
 
-### `Rhum.run`
+* Properties
+    * [Rhum.asserts](#rhumasserts)
+    * [Rhum.mocks](#rhummocks)
+        * [Rhum.mocks.ServerRequest](#rhummocksserverrequest)
+* Methods
+    * [Rhum.afterAll](#rhumafterall)
+    * [Rhum.afterEach](#rhumaftereach)
+    * [Rhum.beforeAll](#rhumbeforeall)
+    * [Rhum.beforeEach](#rhumbeforeeach)
+    * [Rhum.run](#rhumrun)
+    * [Rhum.skip](#rhumskip)
+    * [Rhum.testCase](#rhumtestcase)
+    * [Rhum.testPlan](#rhumtestplan)
+    * [Rhum.testSuite](#rhumtestsuite)
+    
+### Properties
+    
+#### `Rhum.asserts`
 
-Runs your test plan.
+The [asserts](https://deno.land/std/testing/asserts.ts) module from the [testing](https://deno.land/std/testing) module, but attached to `Rhum`.
 
 ```typescript
-Rhum.testPlan(..., () => {
+Rhum.asserts.assertEquals(true, true); // pass
+Rhum.asserts.assertEquals(true, false); // fail
+```
 
-})
-Rhum.run();
+#### `Rhum.mocks`
+
+An object of functions to help you mock objects.
+
+##### `Rhum.mocks.ServerRequest`

```

Comment:
Should `Rhum.mocks` be `###` and `Rhum.mocks.ServerRequest` be `####`? Looks a little odd with `Rhum.mocks` being a smaller font size than the method headings

Reply:
Oh right

Reply:
oh wait. no because Rhum.mocks is a subheading of Properties. Properties is ### and Rhum.mocks is ####

Reply:
Ignore me, i can't see.. They are both the same size :S